microsoft-edge/extensions-chromium/developer-guide/native-messaging.md

```diff
@@ -47,11 +52,33 @@ The following is an example **manifest.json** file:
 }
 ```
 
+##### [Manifest V2](#tab/v2)
+
+```json
+{
+    "name": "Native Messaging Example",
+    "version": "1.0",
+    "manifest_version": 3,
```

Comment:
manifest_version should be 2 for Manifest V2

Reply:
changed 3 to 2